### PR TITLE
Add Pulseaudio into the image

### PR DESCRIPTION
**Pull request by iartemenko**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend [hunk 1]
```diff
@@ -5,6 +5,11 @@ IMAGE_INSTALL_append = " \
     xen-xenstat \
 "
 
+# Add Pulseaudio to build
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend]: please add the below to the IMAGE_INSTALL_append above

Comment (iartemenko) [reply recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend]: Done

File: recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend [hunk 2]
```diff
@@ -1,10 +1,10 @@
-#Add Xen to build
+#Add Xen and additional packages to build
 IMAGE_INSTALL_append = " \
     xen-base \
     xen-flask \
     xen-xenstat \
+    pulseaudio \
 "
-
```
Comment (andr2000) [inline recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend]: not a relevant change

Comment (iartemenko) [reply recipes-dom0/dom0-image-weston/files_rcar/meta-xt-images-extra/recipes-graphics/images/core-image-weston.bbappend]: Done

**Comment by andr2000** [comment]
lgtm, rb

**Pull request merged**